sdcm/mgmt/cli.py

```diff
@@ -37,8 +37,8 @@
 STATUS_DONE = 'done'
 STATUS_ERROR = 'error'
 SSL_CONF_DIR = '/tmp/ssl_conf'
-SSL_USER_CERT_FILE = SSL_CONF_DIR + '/db.crt'
-SSL_USER_KEY_FILE = SSL_CONF_DIR + '/db.key'
+SSL_USER_CERT_FILE = SSL_CONF_DIR + '/test.crt'
```

Comment:
should be imported from the right place ? to avoid defining multiple times ?

Reply:
i.e. form the `sdcm.provision.helpers.certificates` ?

Reply:
Performed tidying up for the cert names here and in a few other places.